mix.exs

```diff
@@ -43,9 +43,12 @@ defmodule CircuitsSim.MixProject do
 
   defp deps() do
     [
-      {:circuits_i2c, github: "elixir-circuits/circuits_i2c", branch: "configurable-backend"},
-      {:circuits_spi, github: "elixir-circuits/circuits_spi", branch: "configurable-backend"},
+      {:circuits_i2c,
+       github: "elixir-circuits/circuits_i2c", branch: "configurable-backend", override: true},
+      {:circuits_spi,
+       github: "elixir-circuits/circuits_spi", branch: "configurable-backend", override: true},
       {:circuits_gpio, github: "elixir-circuits/circuits_gpio", branch: "configurable-backend"},
+      {:bmp280, path: "../bmp280"},
```

Comment:
I really think that the `override: true` above should have made this work. Anyway, I limited the dependency to only be included for dev and test, since when I eventually push to prod, I wouldn't want it being brought in automatically.

```suggestion
      {:bmp280, "~> 0.2.12", only: [:dev, :test]},
```